packages/fuels-core/src/types/wrappers/input.rs

```diff
@@ -16,6 +16,7 @@ pub enum Input {
         resource: CoinType,
         code: Vec<u8>,
         data: Vec<u8>,
+        gas_used: u64,
```

Comment:
this is a breaking change unless we annotated this previously with the `non_exhaustive` attribute.

Reply:
Added

Reply:
adding `non_exhaustive` also makes this change a breaking change, but prevents breaking changes in the future, was that intended?

Reply:
in any case, having `non_exhaustive` only on one enum variant makes it a little annoying for the consumer of this library, so perhaps we can just annotate the whole enum with it?